### DISCUSSION TITLE
iPad freezes when it is rotated or tilted and QField in edit mode

### DISCUSSION THREAD

**Discussion by ranleu** [opengisch/QField#5819] [Bugs]
Hi, 

at our company we have been using a QField project to collect data on the field with several iPads 10.th generation with iPadOS 17.7.1
and have CONSISTENTLY noticed that when in Editing Mode in QField we cannot tilt the tablet too much or rotate it because otherwise it freezes for some minutes. 

Because of this, we need to keep the tablets orientation (vertical or horizontal) which we had, when turning on the Editing Mode to record data efficiently. This is particularly limiting when taking pictures.

I have tested the same QField projects on Android devises with exactly the same QField version and the issue does not come up. 

Neither tablet nor QField crash completely, but the issue makes field work acquard and inneficient. 

There might be a related problem with the azimuthal orientation (@position_orientation) which does not adjust properly when rotating the iPad. It adds/substracts 90 degrees to the azimuth even recording values above 360! I will write separate discussion on this.

**Comment by nirvn** [comment]
@ranleu , this is both surprising and interesting. A couple of follow up questions:
- Do you see this behavior when positioning is off too or only when positioning is active?
- Do you see this behavior when editing the sample bees project shipped with QField?

**Reply by ranleu** [reply]
_Do you see this behavior when editing the sample bees project shipped with QField?_
Yes, it is the same. I even managed to make QField crash from too much rotating/tilting.

_Do you see this behavior when positioning is off too or only when positioning is active?_
It ocurrs in both cases.

**Comment by ranleu** [comment]
Please reactivate the report. I did reply to it.
